### PR TITLE
Vsts matrix

### PR DESCRIPTION
**Pull request by awashbrook**

### CODE REVIEW

**Comment by callumacrae** [comment]
I'm confused, what is this PR doing?

File: .travis.yml [hunk 1]
```diff
@@ -8,4 +8,4 @@ node_js:
   - '0.12'
   - '0.10'
 after_script:
-  - npm run coveralls
+  - npm test
```
Comment (demurgos) [inline .travis.yml]: We already have `language: node_js`, it implies that `script` is `npm test`. This change causes `npm test` to be executed twice.
(Add it removes coverage results)

File: README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@
 [![NPM version][npm-image]][npm-url] [![Downloads][downloads-image]][npm-url] [![Build Status][travis-image]][travis-url] [![AppVeyor Build Status][appveyor-image]][appveyor-url] [![Coveralls Status][coveralls-image]][coveralls-url] [![OpenCollective Backers][backer-badge]][backer-url] [![OpenCollective Sponsors][sponsor-badge]][sponsor-url] [![Gitter chat][gitter-image]][gitter-url]
 
 
-## What is gulp?
+## What is gulp??
```
Comment (demurgos) [inline README.md]: I'm not sure this change is needed.

**Comment by stevelacy** [comment]
None of these changes were requested and none are needed.
We already have travis for running builds, and circle ci is not needed.

**Pull request closed without merging**